### ISSUE TITLE
Validation loss is much much higher than the training loss

### ISSUE THREAD

**Issue by samin9796** [SeanNaren/deepspeech.pytorch#462]
I calculated the validation loss the same way training loss is calculated. Here are the training losses and validation losses we got:

Epoch: 1 Train loss: 212.15943628796458 Val loss : 10288.564560650186 WER: 96.91021459467783 CER: 44.74356377635287

Epoch: 2 Train loss: 89.58623892059244 Val loss : 10227.573487959355 WER: 58.73601914974033 CER: 18.812572980570685

Epoch: 3 Train loss: 60.892252519350016 Val loss : 10225.563025391713 WER: 50.27471758332517 CER: 15.429536961642976

Epoch: 4 Train loss: 46.70684750635436 Val loss : 10214.706202721543 WER: 42.28271064014446 CER: 12.212682544882627

Epoch: 5 Train loss: 37.22001801747798 Val loss : 10211.416837764476 WER: 40.697397707070564 CER: 11.753695076810564

Epoch: 6 Train loss: 30.346706405770977 Val loss : 10204.536970720525 WER: 37.394487450480845 CER: 10.663761263986101

Epoch: 7 Train loss: 24.89854395755072 Val loss : 10199.22469243175 WER: 34.2592074135252 CER: 9.627182280487357

Epoch: 8 Train loss: 20.107685619044393 Val loss : 10197.79249107174 WER: 34.25710766129597 CER: 9.482274642114836

Epoch: 9 Train loss: 16.216724386517882 Val loss : 10193.136769589973 WER: 32.41702479107465 CER: 9.165883902610105

This is train.py https://ideone.com/WQFOKo
This is test.py https://ideone.com/IXzCLP

**Comment by xieyidi** [comment]
hello, i found the train loss increase within an epoch, may i konw the reason。
